site/release-process.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Livy - Release Process
+tagline: Release Process
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+{% include JB/setup %}
+
+# Preparing Livy Releases
+
+*Note: This document is a work in progress and is not guaranteed to be a comprehensive release guide*
+
+## Overview
+
+The release manger for a Apache Livy release is in charge of:
+
+1. Cut and update branches and tags
+1. Create and publish release distros and artifacts to staging locations
+1. Call a vote to authorize release
+1. Publish the release (once a vote passes)
+1. After the release is complete
```

Comment:
Is this sentence finished?

Reply:
This is actually the Table of Contents, thats the name of the last section. I did think of rewording it, but decided it wasn't too important for a initial draft.

Reply:
Ohh, I see, didn't notice it. That's OK for me.